decidim-core/app/services/decidim/send_push_notification.rb

```diff
@@ -36,9 +41,18 @@ def perform(notification)
 
     private
 
-    def notification_params(notification)
+    def message_params(notification, title = nil)
+      case notification
+      when Decidim::PushNotificationMessage
+        notification_params(notification, title)
+      else # when Decidim::Notification
+        notification_params(Decidim::PushNotificationPresenter.new(notification))
+      end
+    end
+
+    def notification_params(notification, title = nil)
       {
-        title: notification.title,
+        title: title.nil? ? notification.title : title,
```

Comment:
I would use the `.presence` method 
```suggestion
        title: title.presence || notification.title,
```

Reply:
While writing this I thought “There should be a more idiomatic way of saying this” 😅